### DISCUSSION TITLE
Unable to see transactions in firefly-iii from ING Netherlands (NL) CSV file

### DISCUSSION THREAD

**Discussion by gangadharjannu** [firefly-iii/firefly-iii#8378] [General]
Hello, 

I found firefly while searching for DIY personal finance management software. 

I am trying to setup in my local using docker using https://docs.firefly-iii.org/how-to/data-importer/installation/docker/

I am able to see firefly and data importer tools, I also setup oAuth for data importer.

So far good but data importing part I think I am missing something. 

Here is what I did, 

I downloaded transactions in CSV file from ING Netherlands, uploaded in data importer tool, 
while trying to import CSV file, nothing visible in firefly UI (I refreshed multiple times but I couldn't see the uploaded transactions)

I tried mapping the roles with no success.

If someone has already done the same process (ING Netherlands transactions in CSV file and imported using data importer), Could you share your configuration or help me ?

I strongly believe I'm missing something but I re read documentation multiple times but not able to figure it out.

I am attaching the sample CSV with dummy data that I'm using to import .

Thanks

[NL01INGB0555566677_06-10-2023_31-12-2023 copy.csv](https://github.com/firefly-iii/firefly-iii/files/13923635/NL01INGB0555566677_06-10-2023_31-12-2023.copy.csv)

**Comment by JC5** [comment]
You could try the example configuration here: https://github.com/firefly-iii/import-configurations/tree/main/nl/ing

If it fails, there should be logs. What do the data importer logs say? See also: https://docs.firefly-iii.org/how-to/general/debug/

**Reply by gangadharjannu** [reply]
Finally I am able to to see the transactions, I think I missed doing the steps in https://github.com/firefly-iii/import-configurations/tree/main/nl/ing

Thank you so much for the prompt response.
